bind/gengo.go

```diff
@@ -8,10 +8,40 @@ import (
 	"fmt"
 	"go/token"
 	"go/types"
+	"runtime"
 	"strings"
 )
 
 const (
+	cgo_check_import = `"strconv"
+	"strings"
+	"os"
+`
+	cgo_check= "cgo_check()"
+	cgo_checkDef = `
+func cgo_check() {
```

Comment:
for consistency with other `gopy` internal functions, please call it `cgopy_check`.